pkgs/by-name/zc/zcool-qingke-huangyou/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  unzip,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "zcool-qingke-huangyou";
+  version = "1.000";
+
+  src = fetchFromGitHub {
+    owner = "googlefonts";
+    repo = finalAttrs.pname;
+    rev = "c9dac424b0a9f47d3b113cff4a4922f632d82c94";
+    hash = "sha256-xIIDP8gCtwNtY6AReeuLZSbnDXczS5ycObP3EKxk+hU=";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/fonts/truetype/
+    install -m644 $src/fonts/*.ttf $out/share/fonts/truetype/
+    runHook postInstall
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    install -Dm644 fonts/*.ttf -t $out/share/fonts/truetype/
    runHook postInstall
  '';
```
1. After unpacking, the CWD is the root of the source, copied into the sandbox. Use relative paths
2. `install -D FILE [FILE ...] -t DIR` both creates directories and installs the file